test/index.test.js

```diff
@@ -95,3 +96,12 @@ test("should handle User type and isAdult function", async (t) => {
 	const { code } = transformSync(inputCode);
 	t.assert.snapshot(code);
 });
+
+test("should handle empty source code", (t) => {
+	assert.strictEqual(transformSync().code, "");
+	assert.strictEqual(transformSync({}).code, "");
```

Comment:
I think it should throw for incorrect types, or use `ToString` to convert the type instead of implicit coercion to empty string.

Reply:
The `ToString` operation is well established and presents in many APIs, like https://github.com/nodejs/node/blob/main/lib/internal/url.js#L809

Reply:
So SWC is already performing toString, which causes `assert.strictEqual(transformSync({}).code, "");` to fail because it is interpreted as `[object Object]` and makes swc throw